index.js

```diff
@@ -21,21 +25,53 @@ const deleteItem = (elem) => {
     const items = document.querySelectorAll('.item')
     const itemsCount = items.length
 
-    let myMap = new Map()
-    myMap.set(0, 'two-items')
-        .set(1, 'three-items')
-
-    for (let item of items) {
-        if (!itemsCount) {
-            document.getElementById("portfolio").style.display = 'none'
-            break
-        } else if (myMap.has(itemsCount % 2)) {
-            item.classList.remove('two-items', 'three-items')
-            item.classList.add(`${myMap.get(itemsCount % 2)}`)
-        }
+    const removeClasses = (element) => {
+        return element.classList.remove('two-items', 'three-items')
     }
-}
 
+    if (!itemsCount) {
+        document.getElementById("portfolio").style.display = 'none'
+        return 0
+    } else {
+        let myMap = new Map()
+        myMap.set(2, 'two-items')
+            .set(3, 'three-items')
+
+        myMap.forEach((value, key) => {
+            if (itemsCount % key === 0) {
+                items.forEach(item => {
+                    removeClasses(item)
+                    item.classList.add(value)
+                })
+            }
+        })
+    }
+}
 
-//other variant example
 document.getElementById('openSidebar').onclick = openSidebar
+
+
+function showInfo(event) {
+    const par = document.createElement('p')
```

Comment:
As an option, you could add `<p>` into markup, find it and add or delete inner text. Could be helpful, if you had styles for `<p>` (no need to add styles through `.style` property)

Reply:
@silchencko Ok, got it, thanks! I just decided to practice with removing the items. Tell me please would I better change the code following this comment?

Reply:
It doesn't really matter. I think which `html` stuff is possible with `html`, should rather be done with `html`. In this case, it's not a big deal. It's up to you.